src/content/docs/fr/guides/integrations-guide/sitemap.mdx

```diff
@@ -401,6 +401,22 @@ Le plan du site qui en résulte ressemble à ceci :
 ...
 ```
 
+### xslURL
+L'URL d'une feuille de style XSL pour styliser et embellir votre sitemap.
+La valeur définie peut-être soit un chemin relatif à votre URL configurée `site` pour une feuille de style locale, soit un lien URL absolu vers une feuille de style externe.
```

Comment:
Nit for the empty line. The wording also seems a bit strange to me:
* "à votre URL configurée `site`" sounds weird
* "lien URL": I think in French we use either "lien" or "URL", I don't think I've ever read "lien URL".

What do you think of:
```suggestion

La valeur définie peut être soit un chemin relatif à l'URL configurée pour votre `site` dans le cas d'une feuille de style locale, soit une URL absolue vers une feuille de style externe.
```